app/[locale]/gallery/[gallery]/[asset]/layout.tsx

```diff
@@ -0,0 +1,38 @@
+import { FunctionComponent, PropsWithChildren } from "react";
+import { getAssetBreadcrumb, getRecentAssets } from "@/lib/api/galleries/asset";
+import { addLocaleUriSegment, useTranslation } from "@/lib/i18n";
+import Container from "@rubin-epo/epo-react-lib/Container";
+import Stack from "@rubin-epo/epo-react-lib/Stack";
+import Buttonish from "@rubin-epo/epo-react-lib/Buttonish";
+import Breadcrumbs from "@/components/page/Breadcrumbs";
+
+export async function generateStaticParams({
+  params: { locale, gallery },
+}: GalleryProps) {
+  return getRecentAssets(locale, gallery);
```

Comment:
generates for all assets right now, can be modified if this adds too much to build time